File: app/src/main/java/cu/axel/smartdock/fragments/PreferencesFragment.java
package cu.axel.smartdock.fragments;

import android.os.Bundle;

import cu.axel.smartdock.R;

import androidx.preference.PreferenceFragmentCompat;

public class PreferencesFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle arg0, String arg1) {
        setPreferencesFromResource(R.xml.preferences_main, arg1);
    }

}

File: app/src/main/java/cu/axel/smartdock/fragments/PreferencesFragment.kt
package cu.axel.smartdock.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import cu.axel.smartdock.R

class PreferencesFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(arg0: Bundle?, arg1: String?) {
        setPreferencesFromResource(R.xml.preferences_main, arg1)
    }
}